### DISCUSSION TITLE
New release for MISE support?

### DISCUSSION THREAD

**Discussion by J3m5** [topgrade-rs/topgrade#792] [General]
Hey there,

First, I'd like to thank you for your work on Topgrade, it's awesome!

Is there any chance we could get a new release with the MISE support from [PR #757](https://github.com/topgrade-rs/topgrade/pull/757)? 
It would be awesome to have it in the next update

Thanks

**Comment by SteveLauC** [comment]
Sure, I will do a release in the near future, though I cannot give you an accurate time:<

**Comment by rubiin** [comment]
+1
i do have custom script for mise step so no rush

**Comment by SteveLauC** [comment]
Hi, v15.0.0 has been released.

**Comment by rubiin** [comment]
It would be nice to support mise self_update on the mise step
